dotnet/src/Azure.Iot.Operations.Connector/ConnectorWorker.cs

```diff
@@ -297,15 +304,7 @@ public async Task ForwardSampledDatasetAsync(Asset asset, AssetDatasetSchemaElem
 
                     string stateStoreKey = destination.Configuration.Key ?? throw new AssetConfigurationException("Cannot publish sampled dataset to state store as it has no configured key");
 
-                    ulong? ttl = destination.Configuration.Ttl;
-                    StateStoreSetRequestOptions options = new StateStoreSetRequestOptions();
```

Comment:
After reading some operator-side documentation, I noticed that this field is for configuring MQTT message expiry intervals, not state store key expiry times